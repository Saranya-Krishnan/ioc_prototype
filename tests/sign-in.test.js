
import React from 'react';
import SignIn from '../views/src/components/sign-in-form';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router'

test('it displays the sign in form', () =>{
    const clickTest = () => {
        return true;
    };
    const component = renderer.create(
        <MemoryRouter>
            <SignIn onClickSubmit={function(){return 'test';}} Item={clickTest}></SignIn>
        </MemoryRouter>
    );
    let c = component.toJSON();
    expect(c).toMatchSnapshot();
});