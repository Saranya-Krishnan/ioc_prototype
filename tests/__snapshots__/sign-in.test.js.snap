// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it displays the sign in form 1`] = `
<div>
  <form
    className="ui form"
    onSubmit={[Function]}
  >
    <div
      className="field"
    >
      <label>
        Email
      </label>
      <input
        name="email"
        onChange={[Function]}
        placeholder="Email"
        value=""
      />
    </div>
    <div
      className="field"
    >
      <label>
        Password
      </label>
      <input
        name="password"
        onChange={[Function]}
        placeholder="Password"
        type="password"
        value=""
      />
    </div>
    <button
      className="ui button"
      onClick={[Function]}
      tabIndex={undefined}
      type="submit"
    >
      Submit
    </button>
  </form>
  
</div>
`;
